src/java/org/apache/cassandra/auth/CassandraRoleManager.java

```diff
@@ -827,4 +853,52 @@ public void setInvalidClientDisconnectMaxJitterMillis(long duration)
         this.invalidClientDisconnectMaxJitterMillis = duration;
         scheduleDisconnectInvalidRoleTask();
     }
+
+    private static final ColumnSpecification GENERATED_PASSWORD_METADATA = new ColumnSpecification(SchemaConstants.AUTH_KEYSPACE_NAME,
+                                                                                                   "generated_password",
+                                                                                                   new ColumnIdentifier("generated_password", true),
+                                                                                                   UTF8Type.instance);
+
+    private static final ColumnSpecification GENERATED_ROLE_NAME_METADATA = new ColumnSpecification(SchemaConstants.AUTH_KEYSPACE_NAME,
+                                                                                                    "generated_role_name",
+                                                                                                    new ColumnIdentifier("generated_role_name", true),
```

Comment:
For consistency, I'd leave this as `generated_name` or change the previous one to `generated_role_password`